src/main/java/org/candlepin/util/Util.java

```diff
@@ -472,6 +472,61 @@ public static <T> boolean collectionsAreEqual(Collection<T> c1, Collection<T> c2
         return true;
     }
 
+    /**
+     * Compares to lists for equality without using the List.equals(...) method. This is primarily intended
```

Comment:
Typo: `Compares to lists` to `Compares two lists`

Reply:
Fixed